src/main/java/cms/gov/madie/measure/services/MeasureService.java

```diff
@@ -589,4 +591,105 @@ private void updateOrganizationName(Measure measure) {
               });
     }
   }
+
+  public MeasureSet associateCmsId(String username, String qiCoreMeasureId, String qdmMeasureId) {
+    if (qiCoreMeasureId == null || qdmMeasureId == null) {
```

Comment:
this works but `StringUtils.isBlank(qiCoreMeasureId) || StringUtils.isBlank(qdmMeasureId)` would be better

Reply:
I changed for the rest for them I think i missed this. thank you I'll change it